### ISSUE TITLE
Usare titolo dei capitoli per i libri anziché il loro id

### ISSUE THREAD

**Issue by cathartyc** [toto04/webeep-sync#110]
Attualmente, i libri vengono sincronizzati inserendo in una cartella tante sottocartelle quanti sono i capitoli, nominati con l'ID corrispondente su WeBeep.

![image](https://github.com/toto04/webeep-sync/assets/67878876/246acb14-f3c4-48f6-b65d-1c7cb7772306)

Sarebbe più comodo trovare il materiale se venisse invece usato il titolo dei capitoli stesso, che da una rapida ricerca nel sorgente della pagina risulta comparire in `<div class="box py-3 generalbox book_content"><div class="rui-title-container"><h3 class=" rui-main-content-title rui-main-content-title--h3">TITOLO DEL CAPITOLO</h3></div>`.

**Comment by toto04** [comment]
mhh questo è particolare, l'app non fa scraping da nessuna parte per quanto riguarda il recupero delle informazioni di file e corsi, usa le API rest di moodle, le cartelle vengono create direttamente usando il path come descritto su moodle appunto, puoi mandarmi il codice del corso così do un'occhiata?

**Comment by cathartyc** [comment]
Certo, il corso nello specifico è `095946 - ADVANCED ALGORITHMS AND PARALLEL PROGRAMMING (FERRANDI FABRIZIO)`.
